lib/deliver/app_screenshot.rb

```diff
@@ -85,7 +91,12 @@ def self.calculate_screen_size(path)
 
       raise "Could not find or parse file at path '#{path}'" if size.nil? or size.count == 0
 
+      # the iOS TV clashes with the 6+ :-/
+
       devices = {
+        ScreenSize::IOS_TV => [
+          [1920, 1080]
+        ],
```

Comment:
so a landscape iPhone 6S resolution _is the same_ as an Apple TV screenshot. I'm not sure this method of keeping track of the platform by the size will work anymore.